tests/test_suites/ShortSystemTests/test_many_parallel_reads.sh

```diff
@@ -0,0 +1,67 @@
+timeout_set 2 minutes
+
+CHUNKSERVERS=12 \
+    USE_RAMDISK=YES \
+    MOUNT_EXTRA_CONFIG="readaheadmaxwindowsize=256,maxreadaheadrequests=30,readworkers=100" \
+    MASTER_CUSTOM_GOALS="8 ec_8_4: \$ec(8,4)"
+    setup_local_empty_saunafs info
+
+cd ${info[mount0]}
+
+# For LongSystemTests
+# cat <<FILES > file_sizes
+# 7676887040 6292065 925696 430500961 2220641 434638896 2417378 289408097 20972129 958554721 2423010 197028 745144897 696081580 304088161 6472801 74980193 614400 1806913 3981889 8306688 8306688 8306688 8306688 925696 925696 925696 925696 10486369 1384448 7533121 8306688 1384448 8306688 8306688 8306688 7299072 966656 966656 8306688 8306688 3801088 925696 925696 8306688 1188961 8306688 34537472 925696 6292065 34537472 34537472 34537472 33185792 8306688 8306688 8306688 8306688 8306688 5701632 8306688 23446 8306688 8306688 8306688 3801088 3801088 8306688 925696 33185792 925696 7691 8389217 3653632 3653632 8306688 794821729 433977482 8389217 5005889 8306688 8306688 8389217 6292065 10486369 6292065 6292065 490734689 861930593 824345185 8389217 12583521 8389217 6292065 10379264 10379264 10379264 10379264 8306688 10379264 8306688 925696 935330913 8389217 196608
+# FILES
+
+output_dir="random_files"
+mkdir -p "$output_dir"
+touch control.files.saunafs
+
+saunafs setgoal -r ec_8_4 "$output_dir"
+saunafs setgoal -r ec_8_4 control.files.saunafs
```

Comment:
Recursive option is not useful in case of files.
```suggestion
saunafs setgoal ec_8_4 control.files.saunafs
```

Reply:
Done.